src/components/Cards/CollectionCard.vue

```diff
@@ -0,0 +1,248 @@
+<template>
+  <SmartLink :link="toCollection">
+    <AspectRatioCard :aspectRatio="aspectRatio" :id="`popover-target-${nid}`" class="card">
+      <template #header>
+        <div class="collection-card-title">
+          {{ title }}
+        </div>
+      </template>
+        <img
+          :src="getImage(puzzleList[0])"
+          style="width: 80%; margin: auto"
+          class="scalable"
+        />
+      <template #footer>
+        <div style="width: 100%;" class="d-flex justify-content-between" v-if="$slots.buttons">
+          <slot name="buttons" />
+        </div>
+        <b-row class="mb-2" style="margin-top: 10px">
+          <b-col cols="6">
+            <div class="left-col">
+              <slot name="left-icon">
+                <img
+                  :src="founder_picture"
+                  alt="founder image"
+                  class="icon"
+                  id="avatarimage"
+                />
+              </slot>
+              {{ founder_name }}
+            </div>
+          </b-col>
+          <b-col cols="6">
+            <div class="right-col">
+              <slot name="right-icon">
+                <img
+                  src="@/assets/noun_puzzle.svg"
+                  alt="collection size"
```

Comment:
Maybe "number of puzzles" is better? Also, please add tooltips for these icons like are added on the puzzle cards now!